src/components/App.tsx

```diff
@@ -1,8 +1,28 @@
 import React from 'react';
+import Products from './Products';
```

Comment:
Something I tend to do is create [barrel files](https://hackernoon.com/react-project-architecture-using-barrels-d086146eb0f6) for organization - typically for components, services, directives, etc.  It reduces the number of import statements.

Reply:
I like that a lot!  Let me build out those page-level components. I can see them being refactored into a "page barrel"

`<page.Cart />`
`<page.ThankYou />`

etc.